src/utils.js

```diff
@@ -1,11 +1,19 @@
 // jarvis, just another rudimentary verbal interface shell
-// converts 'hello "John Doe"' to ['hello', 'John, Doe']
 const fs = require("fs");
 const path = require("path");
+
+// checks whether the given argument is in the format of a stringified JSON
+const isJson = (argument) => {
+  return /<({.+})>/.test(argument);
+}
+
+// converts 'hello "John Doe" <{"name": "JARVIS"}>' to ['hello', 'John Doe',  <{"name": "JARVIS"}>]
 const tokenize = line => {
-  const tokens = line.match(/"([^"]+)"|\S+/g);
+  const tokens = line.match(/"?<({.+})>"?|"([^"]+)"|\S+/g);
```

Comment:
what's the purpose of the change?

Reply:
As mentioned above, parsed string is wrapped by  "`<>`" to make it more robust while tokenizing.